edn/src/symbols.rs

```diff
@@ -278,28 +278,17 @@ impl Display for NamespacedKeyword {
     }
 }
 
+#[macro_export]
 macro_rules! ns_keyword {
```

Comment:
Is it not possible to use the macro you introduced in the previous part?  Is there something about cross-crate macro usage happening?  Is it awkward in tests?